torchtune/datasets/multimodal/_llava_instruct.py

```diff
@@ -54,6 +55,8 @@ class LlavaInstructToMessages(Transform):
         new_system_prompt (Optional[str]): if specified, prepend a system message. This can
             serve as instructions to guide the model response. Setting this will OVERRIDE any system
             messages already present in the dataset. Default is None.
+        images_dir (str): path to the directory containing the images. User is expected to download the COCO dataset.
```

Comment:
not sure if we should have a default. this should be explicitly defined by the user, otherwise they may use the builder/transform and forget to specify this and get confused when it break

Reply:
I like the default for something like this where it's for a very specific dataset.